src/tests/Loader/ContextualReflection/ContextualReflection.cs

```diff
@@ -724,6 +726,19 @@ void TestActivatorCreateInstance(bool isolated)
             }
         }
 
+        void TestDefineDynamicAssembly(bool isolated)
```

Comment:
Nit picking: Should also add another one for the second overload of `DefineDynamicAssembly`.
And then yet another one for the `GetDynamicMethodsModule` - although I don't know how to hit that one...

Reply:
Added tests for `DynamicMethod`. Modified the `DynamicMethod.Init(...)` logic to support creation in various ALCs